### DISCUSSION TITLE
How to set favicon for web after adding expo router

### DISCUSSION THREAD

**Discussion by blakevidebeck** [expo/router#315] [Q&A] [answered]
I have made a new project using expo sdk 48 and the favicon for web view was working correctly.

After adding expo router the favicon stopped working.

Is there a new way to set the favicon after adding expo-router?

**Comment by manuelmaceira** [comment]
According to https://github.com/expo/router/issues/348#issuecomment-1456648645 you should be able to put it in 'public/favicon.ico' but it's not working for me

**Comment by EvanBacon** [comment, marked as answer]
[Supported in v49](https://github.com/expo/expo/issues/21853#issuecomment-1636300941)
